mubench.pipeline/requirements.py

```diff
@@ -98,11 +98,20 @@ def _get_container_cpu_count(self):
             with open("/sys/fs/cgroup/cpu/cpu.cfs_quota_us") as file:
                 quota = int(file.readline())
 
+            """
+                cpu-quota is the guaranteed microseconds of CPU time the container will get.
+                cpu-period is the scheduler period, which defaults to one second in microseconds.
+                The actual access to CPUs can be calculated using 'cpu-quota / cpu-period'.
+                Using '--cpus=X', docker sets the cpu-period to 100000 and cpu-quota to 100000 * X.
+                The default cpu-quota is -1, hence we check the host sytem in that case.
+                Example: '--cpus=1.5' means '--cpu-period=100000 --cpu-quota=150000'.
+            """
             no_limit = quota < 0
             if no_limit:
                 return self._get_normal_cpu_count()
             else:
-                return quota / 100000
+                cpu_period = 100000
```

Comment:
Shouldn't we then retrieve the `cpu_period` also from the docker environment? It seems like a user could set it to a different value on the command line, which would mess up our logic here...